### PR TITLE
feat: update github code links in manual to version branch

### PR DESCRIPTION
**Pull request by trivikr**
Fixes: https://github.com/denoland/deno_website2/issues/1527

### CODE REVIEW

File: components/Manual.tsx [hunk 1]
```diff
@@ -495,7 +495,11 @@ function Manual() {
                     </svg>
                   </a>
                   <Markdown
-                    source={content.replace(/\$STD_VERSION/g, stdVersion)}
+                    source={
+                      content
+                        .replace(/\$STD_VERSION/g, stdVersion)
+                        .replace(/deno\/blob\/master/g, `deno/blob/v${version}`)
```
Comment (lucacasonato) [inline components/Manual.tsx:501]: I would rather we introduce a `$CLI_VERSION` string to replace, and use that in the manual. WDYT?

Comment (trivikr) [reply components/Manual.tsx]: I initially had thought about it after seeing `$STD_VERSION` in documentation - but this was an easier fix.

Comment (trivikr) [reply components/Manual.tsx]: Having said that, this fix will replace all links to master with `${version}` and may introduce bugs. Will replace `$CLI_VERSION` instead.

Comment (trivikr) [reply components/Manual.tsx]: Updated in [`b5cbb74` (#1528)](https://github.com/denoland/deno_website2/pull/1528/commits/b5cbb74e7e713d96993aa852554830cecbe478ca)

**Review by lucacasonato — APPROVED** [review]
LGTM, thanks. Now we just have to go through denoland/deno/docs and update the references there.

**Comment by trivikr** [comment]
> LGTM, thanks. Now we just have to go through denoland/deno/docs and update the references there.

Here: https://github.com/denoland/deno/pull/7710

One issue with this is that code links will not be browse-sable from Markdown files.

**Comment by lucacasonato** [comment]
> One issue with this is that code links will not be browse-sable from Markdown files.

:man_shrugging: - that is already the case with `$STD_VERSION`

**Pull request merged**